src/scripts/i18n/list_source_files.py

```
@@ -0,0 +1,113 @@
+#!/usr/bin/env python3
```

Ruff (0.11.9)

Shebang is present but file is not executable

(EXE001)